docs/css/style.css

```diff
@@ -0,0 +1,894 @@
+@font-face {
```

Comment:
does the docs css have to be checked in? don't particularly like compiled styles being part of any css-changing PR, and doesn't seem like the kind of thing we can't just add to the deployment script.